conans/test/integration/toolchains/env/test_virtualenv_powershell.py

```diff
@@ -114,3 +114,53 @@ def test(self):
     assert "MYPS1!!!!" in client.out
     assert "MYVC_CUSTOMVAR1=PATATA1" in client.out
     assert "MYVC_CUSTOMVAR2=PATATA2" in client.out
+
+@pytest.mark.skipif(platform.system() != "Windows", reason="Requires Windows powershell")
+def test_vcvars():
+    client = TestClient()
+    conanfile = textwrap.dedent(r"""
+        from conan import ConanFile
+        from conan.tools.cmake import CMake, cmake_layout, CMakeToolchain, CMakeDeps
+
+        class Conan(ConanFile):
+           settings = "os", "compiler", "build_type", "arch"
+
+           test_requires = [("gtest/1.12.1")]
```

Comment:
You cannot use the real ``gtest`` in tests, you need to create a fake "hello" or "pkg" dependency for testing. 
Most likely the dependency is not critical for the scope of the test, better drop it.